force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js

```diff
@@ -323,16 +326,10 @@ export default class RecurringDonationTable extends LightningElement {
         this.currentRecord = this.data.find((row) => {
             return row.recurringDonation.Id === e.target.getAttribute("data-recordid");
         });
-        if(this.currentRecord.recurringDonation.CommitmentId__c){
-            this.isElevateDonation = true;
-        }else{
-            this.isElevateDonation = false;
-        }
-        if(this.currentRecord.recurringDonation.Day_of_Month__c){
-            this.isInitiallyMonthlyDonation = true;
-        }else{
-            this.isInitiallyMonthlyDonation = false;
-        }
+
+        this.isElevateDonation = this.currentRecord.recurringDonation.CommitmentId__c ? true : false;
+        this.isInitiallyMonthlyDonation = this.currentRecord.recurringDonation.Day_of_Month__c ? true : false;
```

Comment:
Why not check for InstallmentPeriod = "Monthly" to determine if this _was_ monthly?

Reply:
@force2b you are right, I updated it, thanks.